documentatie/gebruikersdocumentatie/installeren-en-starten.md

```diff
@@ -0,0 +1,91 @@
+# Abacus installeren en starten
+
+Het installeren en starten van Abacus kan op verschillende manieren, met elk verschillende doeleinden.
+Hieronder vind je instructies voor de verschillende methodes.
+
+## Linux en macOS
+
+De onderstaande methodes werken waarschijnlijk op alle gangbare (recente) Linux-distributies en macOS, en zijn getest op Ubuntu (22.04 en hoger) en Debian (12/bookworm).
+
+### Methode 1: build artifact downloaden en starten
+
+Deze methode lijkt het meest op het draaien van een productiebuild en is ook het meest eenvoudig. Voer de volgende stappen uit:
+
+1. Ga in de Abacus-repository naar [Actions](https://github.com/kiesraad/abacus/actions)
+2. Klik aan de linkerkant op [Build, lint & test](https://github.com/kiesraad/abacus/actions/workflows/build-lint-test.yml)
+3. Klik aan de rechterkant op *Branch* om op een branch te filteren
+4. Selecteer de meest recente action
+5. Download het bestand *backend-build*
```

Comment:
Zoals besproken: download-icoon staat er alleen als je ingelogd bent op GitHub. Het is niet nodig om lid te zijn van de Kiesraad-organisatie of speciale toegang te hebben tot de repo.

Reply:
```suggestion
5. Download het bestand *backend-build*. Let op: de download is alleen zichtbaar als je bent ingelogd in GitHub.
```